### ISSUE TITLE
Batchnorm1d layer eigen implementation

### ISSUE THREAD

**Issue by TolyaTalamanov** [NeuralDeepWorks/deepworks#28]
Let's start with implementation for `1D` case: https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm1d.html
